mover-rclone/deploysource.yaml

```diff
@@ -4,9 +4,9 @@ kind: VolumeSnapshot
 metadata:
   name: mover-rclone-snapshot
 spec:
-  volumeSnapshotClassName: gp2-csi
+  volumeSnapshotClassName: csi-hostpath-snapclass
```

Comment:
I wonder if we should just completely remove these lines and let the default storageclass create the volume

Reply:
@cooktheryan Done